### PR TITLE
adding Lee 2015

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -3150,4 +3150,17 @@ @article{Ruhlen1998
   pages        = {13994-13996},
   year         = {1998},
   publisher    = {National Academy of Sciences}
-}
\ No newline at end of file
+}
+
+@article{Lee2015,
+  doi          = {10.1371/journal.pone.0128448},
+  url          = {https://doi.org/10.1371/journal.pone.0128448},
```
Comment (LinguList) [inline concepticondata/references/references.bib:3157]: Nitpicking here, but if the URL is the DOI, I'd pass it as DOI only.

Comment (SimonGreenhill) [reply concepticondata/references/references.bib:3157]: ok, so remove 'url'?

**Comment by LinguList** [comment]
Yes.

File: concepticondata/references/references.bib (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/references/references.bib]: done

**Comment by chrzyki** [comment]
I'll have a look at this in a bit!

Concerning quotes: I'm still a bit lost here. Is there a general recommendation on how to handle them? See the discussion here https://github.com/concepticon/concepticon-data/issues/804 and here https://github.com/concepticon/concepticon-data/issues/611.

**Comment by LinguList** [comment]
I am indifferent, but I don't want normal `"` in the tsv file. One can
use starting and ending quotes, but since the tsv is rendered directly
in html via the app, the `&quot;` is quite okay, and also follows
general markdown style, as markdown allows to default to html.

File: concepticondata/conceptlists/Lee-2015-246.tsv [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+Lee-2015-246-66	66	ice	617	ICE
+Lee-2015-246-67	67	if	1459	IF
+Lee-2015-246-68	68	in, inside	1606	INSIDE
+Lee-2015-246-69	69	intestines	1991	INTESTINES
+Lee-2015-246-70	70	knee	1371	KNEE
+Lee-2015-246-71	71	lake	624	LAKE
+Lee-2015-246-72	72	leaf	628	LEAF
+Lee-2015-246-73	73	left	244	LEFT
+Lee-2015-246-74	74	leg	1297	LEG
+Lee-2015-246-75	75	lightening	1654	LIGHTNING
```
Comment (chrzyki) [inline concepticondata/conceptlists/Lee-2015-246.tsv:76]: Is the typo in the original source as well?

File: concepticondata/conceptlists/Lee-2015-246.tsv [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+Lee-2015-246-95	95	no, not	2279	NO OR NOT
+Lee-2015-246-96	96	nose	1221	NOSE
+Lee-2015-246-97	97	old	1229	OLD
+Lee-2015-246-98	98	One	1493	ONE
+Lee-2015-246-99	99	One Hundred	1634	HUNDRED
+Lee-2015-246-100	100	One Thousand	1843	THOUSAND
+Lee-2015-246-101	101	other	197	OTHER
+Lee-2015-246-102	102	painful, sick	2103	PAINFUL OR SICK
+Lee-2015-246-103	103	person/human being	683	PERSON
+Lee-2015-246-104	104	rain	2108	RAINING OR RAIN
```
Comment (chrzyki) [inline concepticondata/conceptlists/Lee-2015-246.tsv:105]: Can't be disambiguated using the original source?

**Comment by chrzyki** [comment]
Had a first quick look and looks very fine to me, will have another look in the morning.

**Comment by LinguList** [comment]
I guess, the disambiguation question of rain is something one might want
to ask Lee, but we can also compare with their list on Japanese, as they
most likely used similar sources. Otherwise, I'd leave it at this.

File: concepticondata/conceptlists/Lee-2015-246.tsv (hunk 2 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Lee-2015-246.tsv:76]: yep.

File: concepticondata/conceptlists/Lee-2015-246.tsv (hunk 3 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Lee-2015-246.tsv:105]: Anyone know Korean?


비
--
pi
pi
pi
pi
pi
pi
pi
pi
cul-me-i
pi
hɨl-rim
pi
pi
hɨl-rim
pi

**Comment by LinguList** [comment]
Maybe, it is https://en.m.wikipedia.org/wiki/Rain_(entertainer)HeheBut just checked. It is the noun. Like in Chinese. To rain is like rain falls. 
-------- Ursprüngliche Nachricht --------Von: Simon J Greenhill <notifications@github.com> Datum: 07.04.20  23:19  (GMT+01:00) An: concepticon/concepticon-data <concepticon-data@noreply.github.com> Cc: Johann-Mattis List <mattis.list@lingpy.org>, Comment <comment@noreply.github.com> Betreff: Re: [concepticon/concepticon-data] adding Lee 2015 (#819) 

@SimonGreenhill commented on this pull request.







In concepticondata/conceptlists/Lee-2015-246.tsv:

> +Lee-2015-246-89	89	near	1942	NEAR

+Lee-2015-246-90	90	neck	1333	NECK

+Lee-2015-246-91	91	needle	1382	NEEDLE (FOR SEWING)

+Lee-2015-246-92	92	new	1231	NEW

+Lee-2015-246-93	93	night	1233	NIGHT

+Lee-2015-246-94	94	Nine	1483	NINE

+Lee-2015-246-95	95	no, not	2279	NO OR NOT

+Lee-2015-246-96	96	nose	1221	NOSE

+Lee-2015-246-97	97	old	1229	OLD

+Lee-2015-246-98	98	One	1493	ONE

+Lee-2015-246-99	99	One Hundred	1634	HUNDRED

+Lee-2015-246-100	100	One Thousand	1843	THOUSAND

+Lee-2015-246-101	101	other	197	OTHER

+Lee-2015-246-102	102	painful, sick	2103	PAINFUL OR SICK

+Lee-2015-246-103	103	person/human being	683	PERSON

+Lee-2015-246-104	104	rain	2108	RAINING OR RAIN



Anyone know Korean?

비

pi

pi

pi

pi

pi

pi

pi

pi

cul-me-i

pi

hɨl-rim

pi

pi

hɨl-rim

pi



—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

[

{

"@context": "http://schema.org",

"@type": "EmailMessage",

"potentialAction": {

"@type": "ViewAction",

"target": "https://github.com/concepticon/concepticon-data/pull/819#discussion_r405119750",

"url": "https://github.com/concepticon/concepticon-data/pull/819#discussion_r405119750",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

"@type": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Comment by SimonGreenhill** [comment]
Just realised I should add the Korean glosses too.

**Comment by LinguList** [comment]
My message did not come through, I am afraid: the rain is a noun here, judging from the Korean gloss (to rain means, I think, "rain fall", similar to Chinese as construction).

**Comment by LinguList** [comment]
But that would be the last thing I recommend to do, then we can merge. @CarolinHu, this is an easy list, so we'll do with one reviewer only, and you don't need to feel forced to look at the list now.

**Pull request merged**